blocks/hero-block/features/hero/default.jsx

```diff
@@ -34,20 +41,95 @@ function Hero({ customFields }) {
 		subHeadline,
 		variant = "dark",
 	} = customFields;
-
+	const { arcSite } = useFusionContext();
+	const { fallbackImage, resizerURL } = getProperties(arcSite);
+	const imageId = imageDesktopURL
+		? imageDesktopURL.split("/").pop().split(".").shift()
+		: desktopImgId;
+	const imageMobileId = imageMobileURL
+		? imageMobileURL.split("/").pop().split(".").shift()
+		: imgMobileId;
+	let desktopAuth = useContent(
+		imageId
+			? {
+					source: "signing-service",
+					query: { id: imageId },
+			  }
+			: {}
+	);
+	const imageAuthTokenObj = {};
+	if (desktopAuth?.hash) {
+		desktopAuth[RESIZER_TOKEN_VERSION] = desktopAuth.hash;
+	}
+	if (desktopAuth && !Object.keys(desktopAuth).length) {
+		desktopAuth = imageDesktopAuth;
+	}
+	let mobileAuth = useContent(
+		imageMobileId
```

Comment:
Same as for the desktop image, this should be `!imageMobileAuth && imageMobileId`.

Reply:
Fixed it.